index.html

```diff
@@ -70,39 +66,134 @@ <h2>新規タスクの追加</h2>
             const button = document.getElementById('btn');
             const lists = [];
             const btnWork = document.createElement('button');
+            const jsInput = document.getElementById('js-input');
+            const collect = document.getElementById('collect');
+            const radioButtonAll = document.getElementById('all');
+            const radioButtonWorking = document.getElementById('work');
+            const radioButtonDone = document.getElementById('complete');
+            const stateWork = '作業中';
+            const stateDone = '完了';
+
             button.addEventListener('click', function () {
+                const todo = { id: lists.length + 1, comment: jsInput.value, state: stateWork };
+                lists.push(todo);
+                clearList();
+                createList();
+
+            });
+
+            function clearList() {
+                const list = document.getElementById('add-list');
+                while (list.firstChild) {
+                    list.removeChild(list.firstChild);
+                }
+            }
+
+            function createList() {
+                lists.map((todo) => {
+                    console.log(todo.state);
+                    createTodo(todo, todo.state);
+                });
+            }
+
+            function createListByStatus(status) {
+                lists.map((todo) => {
+                    if (todo.state == status) {
```

Comment:
厳密に比較ができる `===` を利用してください。
```suggestion
                    if (todo.state === status) {
```